packages/stockflux-watchlist/src/components/watchlist/Watchlist.js

```diff
@@ -66,36 +68,22 @@ const Watchlist = () => {
 
   useEffect(() => {
     if (notification.state === 'LAUNCHED') {
-      const alreadyInWatchlist = watchlist.includes(newSymbol);
       notification.populate(
-        <>
-          <div className="notification-icon">
-            <img
-              id="icon"
-              className={alreadyInWatchlist ? 'arrow-up' : 'card-icon'}
-              src={alreadyInWatchlist ? 'ArrowUp.png' : 'CardIcon.png'}
-              alt="Notification Icon"
-            />
-          </div>
-          <div id="notification-content">
-            <p id="title">WATCHLIST UPDATED</p>
-            <hr />
-            <p id="info">
-              <span id="symbol">{newSymbol}</span>
-              <span id="message">{`${
-                alreadyInWatchlist ? ' moved' : ' added'
-              } to the top`}</span>
-            </p>
-          </div>
-        </>
+        <div className="notification">
+          <div className="icon"></div>
+          <p>
+            {newSymbol.symbol}
+            {newSymbol.isNew
+              ? ' has been added to your Watchlist'
+              : ' has been moved to the top of your Watchlist'}
```

Comment:
I wonder if its worth pulling out every instance where we have text like this into a single place? so that way if text ever needs changing it is only in a single place for every place?
Not part of this PR, but one for the backlog, as it could be useful for text like "no search results" etc.

Reply:
Yeah, a very good idea, that would be very nice indeed!